sql/core/src/test/resources/sql-tests/results/timezone.sql.out

```diff
@@ -0,0 +1,121 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 11
+
+
+-- !query
+SET TIME ZONE 'Asia/Hong_Kong'
+-- !query schema
+struct<key:string,value:string>
+-- !query output
+spark.sql.session.timeZone	Asia/Hong_Kong
```

Comment:
This looks like a weird output for `SET TIME ZONE 'Asia/Hong_Kong'`. Shall we add a `Project` node on the `SetCommand` to give better output>

Reply:
What do we expect as output? only keep the value `Asia/Hong_Kong` here?

Reply:
is there any reference?

Reply:
```
postgres=# SET TIME ZONE 'Asia/Hong_kong';
SET
```